### PR TITLE
Update auth_mechanisms.rst

### PR DESCRIPTION
**Pull request by Rello**
note on external storage authentication

### CODE REVIEW

File: admin_manual/configuration_files/external_storage/auth_mechanisms.rst [hunk 1]
```diff
@@ -74,3 +76,16 @@ See :doc:`./sftp` for additional information on how to set up certificate based
...
+.. _considerations_for_shared_storage_label:
+
+Considerations for shared storage
+---------------------------------
+
+Every external storage, which is using user specific authentication, is connected individually.
+Even if several users connect to the same folder, the files are regarded as separate files per user.
+Nextcloud can not recognize if two users access the very same file if they follow individual connections. 
+
+This has an influence on e.g. file locking as a locked individual file is not shown as locked to other users.
```
Comment (juliushaertl) [inline admin_manual/configuration_files/external_storage/auth_mechanisms.rst:89]:
```suggestion
This has an influence on e.g. file locking as a locked individual file is not shown as locked to other users or users cannot collaboratively edit documents.
```

**Review by juliushaertl — COMMENTED** [review]
Small addition to mention further limitations, otherwise looks good 👍

**Comment by Rello** [comment]
@juliushaertl ich glaub du musst nochmal approven nach dem edit?

**Review by juliushaertl — APPROVED** [review]

**Comment by AndyScherzinger** [comment]
@juliushaertl @Rello I guess we best backport this one? As far back as we still publish it I would say.

Can any of you trigger/cover that please? Thanks in advance

**Pull request merged**